.github/workflows/build-wheel-macos-arm64.yaml

```diff
@@ -308,6 +308,13 @@ jobs:
         cmake --build runtime-build --target runner_tests_lightning runner_tests_openqasm
         ./runtime-build/tests/runner_tests_lightning
         ./runtime-build/tests/runner_tests_openqasm
+  
+    # Build OQC-Runtime
+    - name: Build OQC-Runtime
+      run: |
+        	cmake -G Ninja -B oqc-build \
+                -DRUNTIME_BUILD_DIR=$GITHUB_WORKSPACE/runtime-build \
+        	cmake --build oqc-build --target rtd_oqc
```

Comment:
Can we invoke the make command her instead? If possible it's best to only have one copy of the build recipe

Reply:
I was wondering the same but all the other one are using the build recipe, I was thinking there was a reason?